<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QWidget" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2480</width>
    <height>1427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QPushButton" name="clearBtn">
   <property name="geometry">
    <rect>
     <x>2030</x>
     <y>1200</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:20px;
border-radius: 20px;
font: 20pt &quot;나눔스퀘어_ac Bold&quot;;
background-color: rgb(255, 212, 0);
</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="brushBtn">
   <property name="geometry">
    <rect>
     <x>2290</x>
     <y>1030</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>2250</x>
     <y>1200</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:20px;
border-radius: 20px;
font: 20pt &quot;나눔스퀘어_ac Bold&quot;;
background-color: rgb(255, 212, 0);
</string>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
